### DISCUSSION TITLE
Lesson 8 MetaMask Hex data

### DISCUSSION THREAD

**Discussion by warrenbeufesche** [Cyfrin/foundry-full-course-cu#259] [General]
I am on Lesson 8 (2:54:33). I have Chrome MetaMask Ver 20.33. I have looked all over for the Hex data in MetaMask and cannot find it. I see a dialog with Fund at the top, Transaction, Activity Log, Transaction data sections. Is there other information I am missing?

**Comment by ghost** [comment]
I think that feature is off by default, go to Settings => Advanced => Show Hex Data.

**Comment by warrenbeufesche** [comment]
Thanks for the quick reply! I do have Show Hex Data: ON. My MetaMask looks nothing like the video. I am seeing some of the data but not the Hex data. I have Ubuntu 22, Chrome Ver 114, MetaMask Ver 2.33. I am thinking my version is the problem, the video was probably shot on a Mac. I am okay for now though, but I am wondering if this is a version issue?

**Reply by ghost** [reply]
Well, Metamask recently updated their UI but if I try to make a transaction that involves a contract interaction (calling a function) this is what I see:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/109909308/7cf25839-98aa-4c10-99d8-50b80f8f9ce0)

If you don't see anything like it after initiating the tx, then I recommend you going to a Metamask forum e.g. Reddit: https://www.reddit.com/r/Metamask/.

Just remember to **never share** anything private like a Private Key or Secret Phrase.

**Reply by warrenbeufesche** [reply]
Thanks, I will check reddit.
